### PR TITLE
Str limits

### PR DESCRIPTION
**Pull request by bjhall**
Fixes https://github.com/Clinical-Genomics/scout/issues/1425

Screenshot:
image

- [x] code approved by dNil
- [x] tested by bjhall

### CODE REVIEW

File: scout/server/blueprints/variants/templates/variants/str-variants.html [hunk 1]
```diff
@@ -37,15 +37,17 @@
...
+          <th style="width:7%" title="Repeat ID">Repeat locus</th>
+          <th style="width:12%" title="Repeat unit">Reference repeat unit</th>
+          <th style="width:7%" title="ALT">Estimated size</th>
+          <th style="width:7%" title="ReferenceSize">Reference size</th>
+          <th style="width:9%" title="Status">Status</th>
+          <th style="width:7%" title="Max normal">Max normal</th>
+          <th style="width:7%" title="Min pathologic">Min pathologic</th>
+          <th style="width:14%" title="GT">Genotype</th>
+          <th style="width:5%" title="Chromosome">Chr.</th>
+          <th style="width:10%" title="Position">Position</th>
```
Comment (dnil) [inline scout/server/blueprints/variants/templates/variants/str-variants.html]: sum(width) != 100%?

File: scout/server/blueprints/variants/templates/variants/str-variants.html [hunk 2]
```diff
@@ -55,9 +57,9 @@
           {% elif variant.str_status == 'normal' %}
               <tr class="bg-success">
           {% elif variant.str_status == 'full_mutation' %}
-              <tr class="bg-warning">
+              <tr class="bg-danger">
           {% elif variant.str_status == 'pre_mutation' %}
-              <tr style="background-color: #ffff66;">
+              <tr class="bg-warning">
```
Comment (dnil) [inline scout/server/blueprints/variants/templates/variants/str-variants.html]: :+1:

File: scout/parse/variant/variant.py [hunk 3]
```diff
@@ -193,6 +193,16 @@ def parse_variant(variant, case, variant_type='clinical',
     if str_status:
         parsed_variant['str_status'] = str(str_status)
 
+    # str normal max - max number of repeats to call an STR variant normal
+    str_normal_max = variant.INFO.get('STR_NORMAL_MAX')
```
Comment (dnil) [inline scout/parse/variant/variant.py]: Nice! Can we get a test as well?

Comment (dnil) [reply scout/parse/variant/variant.py]: Coveralls says ⛔️. 😉

File: scout/server/blueprints/variants/templates/variants/str-variants.html (hunk 1 above)
Comment (bjhall) [reply scout/server/blueprints/variants/templates/variants/str-variants.html]: I've fixed this!

File: scout/parse/variant/variant.py (hunk 3 above)
Comment (bjhall) [reply scout/parse/variant/variant.py]: I've added tests for STR parsing. I had to add new STR vcfs that were annotated with the latest Stranger version. Hopefully the tests are somewhat meaningful. I basically copied them from the SNV/SV tests.

**Pull request merged**